examples/analyze.py

```diff
@@ -39,38 +39,49 @@
              }
 
 if __name__ == "__main__":
-    # Check that directories exists, create if it doesn't.
+    # Check that directories exists, create if they don't.
     if not os.path.exists(beads_plot_dir):
         os.makedirs(beads_plot_dir)
     if not os.path.exists(gated_plot_dir):
         os.makedirs(gated_plot_dir)
 
-    # Process beads data
+    # Load beads data
     print("\nProcessing beads...")
     beads_data = FlowCal.io.FCSData('{}/{}'.format(directory, beads_file))
     print("Beads file contains {} events.".format(beads_data.shape[0]))
 
-    # Trim
+    # Transform to Relative Fluorescence Units (rfi), commonly known as
+    # arbitrary units (a.u.)
+    beads_data = FlowCal.transform.to_rfi(beads_data,
+                                          sc_channels + fl_channels)
+
+    # Gating
+
+    # Remove first and last events. Transients in fluidics can make the
+    # first few and last events slightly different from the rest.
     beads_data = FlowCal.gate.start_end(beads_data, num_start=250, num_end=100)
+
+    # Remove saturating events in forward/side scatter.
     beads_data = FlowCal.gate.high_low(beads_data, sc_channels)
```

Comment:
do you have specific knowledge of the files acted upon by `analyze.py` which necessitates `gate.high_low()`? In general, I don't think `gate.high_low()` is automatically warranted anymore (e.g. floating point data). might consider adding a check of the data type here unless you already know what these files are. sounded like the examples were gonna change soon anyways? so maybe not a big deal.